plugins/amazonq/codewhisperer/jetbrains-community/tst/software/aws/toolkits/jetbrains/services/codewhisperer/CodeWhispererUserInputTest.kt

```diff
@@ -98,35 +86,4 @@ class CodeWhispererUserInputTest : CodeWhispererTestBase() {
             }
         }
     }
-
-    private fun addUserInputAfterInvocation(userInput: String) {
-        val codewhispererServiceSpy = spy(codewhispererService)
-        val triggerTypeCaptor = argumentCaptor<TriggerTypeInfo>()
-        val editorCaptor = argumentCaptor<Editor>()
-        val projectCaptor = argumentCaptor<Project>()
-        val psiFileCaptor = argumentCaptor<PsiFile>()
-        val latencyContextCaptor = argumentCaptor<LatencyContext>()
-        codewhispererServiceSpy.stub {
-            onGeneric {
-                getRequestContext(
-                    triggerTypeCaptor.capture(),
-                    editorCaptor.capture(),
-                    projectCaptor.capture(),
-                    psiFileCaptor.capture(),
-                    latencyContextCaptor.capture()
-                )
-            }.doAnswer {
-                val requestContext = codewhispererServiceSpy.getRequestContext(
-                    triggerTypeCaptor.firstValue,
-                    editorCaptor.firstValue,
-                    projectCaptor.firstValue,
-                    psiFileCaptor.firstValue,
-                    latencyContextCaptor.firstValue
-                )
-                projectRule.fixture.type(userInput)
-                requestContext
-            }.thenCallRealMethod()
-        }
-        ApplicationManager.getApplication().replaceService(CodeWhispererService::class.java, codewhispererServiceSpy, disposableRule.disposable)
-    }
```

Comment:
uh, is this intentional ?
i saw https://github.com/aws/aws-toolkit-jetbrains/pull/5270/files#diff-f85f84ca7bee9e47b4f711e46fb55f437df3578e10f829b8160211b65a328d53L89 is still calling it tho